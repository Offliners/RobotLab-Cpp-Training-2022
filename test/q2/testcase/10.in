409
jOURNALISM
diAmoNd
beAr
EvENtuAL
MEdaL
WAsTe
GAMBlE
35e85
ComMisSiON
CONSIST
v8ry
MILlIoNAiRE
bestselLer
FuRThERMOrE
LABoRATOry/LaB
b72954
ObjeCTIVe
1n0i2Idate
MILLIGraM
cattle
painter
CREam
peRcent
AIM
13cu9t0
ViRTUOUS
growth
ClEaniNG
RAnK
BEHAVE
boDiLy
ADVERSE
founDeR
AGENDA
ruggeD
RESOnANCE
TrIumpH
TOW
accurate
IdIOt
compaRtment
Drug
TEACH
dIscOurAge
DISCOvERY
TIE
DIsTriBuTE
wAsh
1AI5
FuNny
GarlIc
sterEo
GrAIN
SYNoNym
november
dread
FLeXIBlE
Plainly
aRtiLlErY
TimE
UpBringInG
capitol
inTeRACT
FRUIt
bother
hungeR
LIMITED
Wheel
POPCORN
ENcYClOPedIA
DiscOvER
REFORM
reMAins
mOrTAl
abOriginal
DEPIct
REGIONAL
outdoOr
BaTTLE
cITiZeN
reLIgioN
PAiN
m
mathematics/matH
DiNNeR
13g59
LiMITATIon
aborigine
rEguLATe
tall
im9g9
dYnaMic
broaden
LOOSELY
bReAKFAST
leAF
MUstAcHE
housing
poTeNt
CuRtAIN
duKe
skate
DUMb
Valuable
F2T7l
omen
lip
maGNIfIcENT
toTaLlY
beVERaGE
eel
tRUNK
TRIBUNE
RheTOric
subsEqUeNTlY
horMonE
n
u
anAlYSt
strENgtH
CLAY
apOlOGize
AftermAth
under
BLANKET
obITuAry
DEVICe
aDvicE
fAsTEN
SPeRM
eXplODE
effective
kEen
CoNSoLe
INtERPrEtATiON
fit
eXhIbitioN
wARShIP
dEFEnsIVE
0E69
lESsER
skY
b0484E
beSide
killer
CONFIdENTIAl
noodle
dISINTeRESTEd
c
FOX
inSTrUcTIon
MeET
humid
Fork
COMMENT
TURN
HUMORoUSLY
unlimited
tHEmE
cOffee
hollywooD
conTiNuAL
reSist
EMplOY
EXaMinEe
disappear
ordInaNce
p613o2al
REcESsION
EaSter
enemy
rather
mEAsles
PRogReSs
VENt
eLeCT
dIctaTOr
further
InclInaTion
HONOr
sOCCER
coNsidErAtion
INEFFECTIVE
HOLE
PuRpOSE
peak
CONSENT
BikInI
IDentIficATIOn
STORY
SQUIrREL
CHARMING
SUPPORT
ultimately
CuStomEr
relatively
voice
cOMBINE
proteCTIve
sEcURE
8e67u985
541l6
dELInquent
dETERGENT
SWAy
HEiR
illUstrate
abSoRB
oVERLAp
upSEt
trAy
fULL
oWn
o7d
CApTURe
FlAt
38r319451615
CHOOSE
Loom
radio
SEX
SIghT
outward
CoMParativE
UtMOsT
BLACkBOARd
COiN
squARely
DiSCRIMiNAtE
SILENtLY
ChOp
eXCEpT
Stiff
ReFERENDUM
STOraGE
MODEL
huge
UnCOMFortablE
pROmpt
river
TEnTatIVe
traVeRsE
AMaZING
SiNgER
4979h
CONTACT
DIVersiTY
refreshment
acreage
dar6sAy
GraY
DRapE
rEMedy
BEAR
BENeFIcIAL
V1o949E
close
FOCUS
DISTINCTIVELY
stereotYpe
inclUde
bALD
FICTION
vInEGAr
DAtE
diE
SPARkLE
LECTUrER
o
EarPlUG
foREVer
bass
SAD
rAGE
syMPaTHIZE
tyrAnt
SIXTY
DiFfusiON
ROYALTY
dISco
POLiCy
43A5B832
CRUDe
TeRminal
lENgthy
bInd
tonE
priSon
WHY
eMOtioNaL
PErsUASIVE
FLATtER
INViTe
committEE
ISOLAtE
Radiate
Jaywalk
leT
marvEl
visUALIZE
BaG
strUgGLE
SOBeR
celebration
QUESTIONNAIRE
base
vertical
DASH
return
plaYGRound
CHErISh
HALftVaY
mil344n2
CHAptER
cHAoS
candidatE
HUGe
GASP
iN77BI04n1
Museum
MachINERY
CABIn
soUvenir
NoTIFY
cOLumnisT
truCk
CLEANe3
bEgIN
tie
nONEtHeLESS
sUCh
hANd
cOLLEAGUe
split
evIL
COnTROvERsIal
TEchnoloGy
CALm
STRATEGY
PreSeNCe
loVEr
aRiTHmETIC
SLicE
hABIt
KINDLy
piNt
Bloody
PrOOF
afFord
c
CoNstrUcTI9n
THINK
booT
figHter
rIng
cUb
Head
gloriOus
MIld
brutal
ANALOGy
c
tOFU
hangEr
tORTURE
mAGNiFY
defEnSe
obediEnT
conNect
GOD
odds
rely
aLIKE
812g
lEaRn2d
KETtLE
fronT
blonde
milITARY
UPROOT
WAiT
maKeup
CreATION
LoosEN
outinG
gLobE
CloseT
OfFiCEr
incredible
LaNGUAgE
MaRK
wEsTerN
downtoWN
ceramIc
d5AlECt
toll
ceREaL
AcCOMPLISHMENT
pANel
dolphin
GUARANTEE
cONVENIEnT
PSYChOLOgY
PUrPLE
fOund
DroWsy
tOileT
LoUsy
allocation
