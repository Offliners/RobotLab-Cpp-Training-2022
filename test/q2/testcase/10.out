n. 新聞業
n. 鑽石
n. 熊；魯莽之人
adj. 結果的
n. 獎章；紀念章
n./v. 浪費
v. 賭博；投機
Unknown
n./v. 佣金；委託
v. 組成；存在於…
Unknown
n. 百萬富翁
n. 暢銷品
adv. 此外
n. 實驗室
Unknown
adj. 客觀的
Unknown
n. 毫克
n. 牛；牲口
n. 畫家
n. 奶油；乳脂
n. 百分之…
n./v. 目標；瞄準
Unknown
adj. 品德高尚的
n. 生長；發育
n. 掃除
n./v. 等級；並排
v. 表現；行為舉止
adj. 身體的
adj. 反對的
n. 創立者；締造者
n. 議程
adj. 崎嶇的
n. 共嗚
n./v. 勝利；戰勝
v./n. 拖
adj. 準確的
n. 白癡
n. 分隔間
n./v. 藥品；毒品
v. 教導
v. (n)使洩氣；使沮喪
n. 發現
n. 領帶
v. 分配
n./v. 洗滌
Unknown
adj. 有趣的
n. 大蒜
n. 立體聲；立體音響器材
n. 穀粒；穀物
n. 同義詞
n. ＋一月
n./v. 害怕
adj. 有彈性的
adv. 清楚地
n. 大砲
n./v. 時間
n. 養育
n. 美國州議會大廈
v. 互動
n. 水果；成果
v./n. 打擾；煩惱
n. 飢餓
adj. 被限制的
n./v. 輪子；旋轉
n. 玉米花
n. 百科全書
v. 發現
v. 改革
n. 遺跡
adj. 必死的
adj. adj. 原住民的
v. 描繪
adj. 地區的
adj. 戶外的
n./v. 戰役
n. 公民；市民
n. 宗教
n. 疼痛
Unknown
n. 數學
n. 晚餐
Unknown
n. 限制
n.  原著居民
v. 限制；控管
adj. 高的
Unknown
adj. 動力的；動態的
v. 加寬
adv. 不嚴格地；寬鬆地
n./v. 早餐
n. 葉子；一張或一頁
n. 鬚
n. 住房建築
adj. 有效力的
n./v. 簾；給…裝上簾子
n. 公爵
n./v. 冰鞋；溜冰
adj. 啞的；愚笨的
adj. 珍貴的
Unknown
n. 微兆；預兆
n. 嘴唇
adj. 壯麗的
adj. 完全地
n. 飲料
n. 鰻
n. 樹幹
n. 護民官
n. 修辭學
ad v. 後來
n. 荷爾蒙
Unknown
Unknown
n. 分析家
n. 力氣
n. 黏土；泥土
v. 道歉
n. 餘波
prep./adv. 在…下面
n. 毛毯
n. 訃聞
n. 裝置
n. 勸告；消息
v. 繫緊；集中注意力
n. 精子
v. 爆炸
adj. 有效的
adj. 熱衷的
v. 安慰
n. 解釋；翻譯
adj/n/v. 適合的
n. 展示會
n. 軍艦
adj. 防護的；保衛的
Unknown
adj./adv. 較小的
n. 天空
Unknown
prep. 在旁邊
n. 殺人者
adj. 機密的
n. 麵條
adj. 公正的
Unknown
n. 狐狸
n. 教導；指示
v. 遇見
adj. 潮溼的
n./v. 叉子；分岔
n./v. 評論
n./v. 轉動
adv. 幽默地
adj. 無限的
n. 主題
n. 咖啡
n. 好萊塢
adj. 不間斷的
v. 反抗
v. (n)雇用
n. 應試者
v. 消失
n. 法令；條例
Unknown
n. 經濟衰退
n. 復活節
n. 敵人
adv. 與其不如
n. 麻疹
n./v. 前進
n. 通風孔
v. 選舉
n. 獨裁者
adj./adv./v. 進一步的
n. 傾斜
n./v. 榮譽；使…增光
n. 足球
n. 考慮；動機
adj. 無效力的
n. 孔；漏洞
n. 意圖
n. 山頂
n./v. 同意
n. 三點式泳裝
n. 識別；身分證明
n. 故事
n. 松鼠
adj. 有魅力的
n./v. 支持
adv. 最後
n. 顧客
adv. 相對地
n./v. 聲音；表達
v. 結合
adj. 保護的
adj. 安全的
Unknown
Unknown
adj. 不法行為
n. 洗潔劑
v. (使)搖動
n. 繼承人
v. 圖解；說明?
v. 吸收；理解
v. 部分重疊；部分相同
adj./v. 心煩的；使生氣
n. 盤子
adj. 滿的；完全的
adj./v. 自己的
Unknown
v./n. 捕獲；獲物
adj/adv. 平坦的
Unknown
v. 選擇
v. 隱約的或威脅性的出現
n. 收音機
n. 性(別)
n./v. 視力；看見
adj. 外面的；肉體的
adj. 相比而言的
adj. 極度的
n. 黑板
n. 硬幣
adv. 正直地
v. 辨別
adv. 寂靜地
v./n. 劈；肋排肉
prep. 除了…之外
adj. 硬的
n. 公民投票
n. 儲存
n. 模型；模範
adj. 龐大的
adj. 不舒服的
adj. 促使
n. 河流
adj. 試驗性的
v. 橫越
adj. 令人驚訝的
n. 歌手
Unknown
n./v. 接觸；聯繫
n. 多樣性
n. 精神恢復
n. 英畝數
Unknown
adj./n. 灰色的
v. 覆蓋
n. 治療
n. 熊；魯莽之人
adj. 有益的
Unknown
adj./adv. 接近的
n./v. 焦點；集中
ad v. 有特色地
n. 老套
v. 包括
adj. 禿的
n. 小說；虛構的事
n. 醋
n. 棗；棗椰樹
v. 死亡
n./v. 閃耀
n. 講演者；講師
Unknown
n. 耳塞
adv. 永遠
n. 低音部
adj. 難過
n./v. 發怒
v. 同情
n. 暴君
adj./n. 六十
n. 擴散
n. 皇室成員
n. 迪斯可舞廳/音樂
n. 政策
Unknown
adj. 未加工製造的
n. 終點；總站
adj. 冗長的
v. 捆；綁
n. 音調
n. 監獄
adv./pron./conj.  為什麼
adj. 感情上的
adj. 有說服力的
v. 諂媚；奉承
v. 邀請
n. 委員會
v. 使孤立
v. 發射
v. 不遵守交通規則而穿越馬路
v. 允許；讓
n. 令人驚奇或驚喜的事物
v. 想像
n. 袋子
n./v. 掙扎；努力
adj. 清醒的
n. 慶祝
n. 問卷
n./v. 基礎
adj. 垂直的
v. 猛衝
n./v. 回復；歸還
n. 操場
v. 珍惜
adj./adv. 位於兩地中途的
Unknown
n. 章，回
n. 無秩序狀態
n. 候選人
adj. 龐大的
v. 喘息
Unknown
n. 博物館
n. 機械裝置
n. 客艙
n. 紀念品
v. 通知
n. 專欄作家
n./v. 卡車；用卡車裝運
Unknown
v. 開始
n. 領帶
adv./conj. 但是；仍然
adj./prop. 如此的
n./v. 手；交給
n. 同事
n./v. 裂縫；(使)裂開
adj./n. 邪惡的
adj. 有爭議的
n. 工業技術；(一門)科技
adj./v.  鎮靜的
n. 策略
n. 出席
n. 戀人
adj./n. 算術的
n. 片
n. 習慣
adj. 友善的
n. 品脫(液量名)
adj. 血腥的
n. 證據
v. 類得起；承擔得起
Unknown
Unknown
v. 想
n. (長筒)靴
n. 鬥士；戰鬥機
n./v. 圈；圍住
n. 幼獸
n. 頭
adj. 光榮的；輝煌的
adj. 溫和的；順口的
adj. 殘忍的
n. 相似
Unknown
n. 豆腐
n. 衣架；絞刑執行者
v. 拷打；折磨
v. 擴大；誇大；誇獎
n. 防衛
adj. 順從的；孝順的
v. 連結
n. 上帝；造物主
n. 可能的機會
v. 信賴
adj./adv.  相同的
Unknown
Unknown
n. 水壺
adj./n. 前面(的)
n. 金髮白種人
adj./n. 軍事的；軍隊
v. 根除
n./v. 等候
n. 裝扮；化妝品
n. 創作；萬物
v. 放鬆
n. 遠足
n. 地球；球狀物
n. 壁櫥；衣櫥
n. 軍官；官員
adj. 驚奇的
n. 語言
n./v. 痕跡；記號
adj./n. 西方世界
adv./adj./n. 在城市的商業區；鬧區
adj. 陶器(藝)的
Unknown
n. 通行費
n. 玉蜀黍薄片
n.  才藝；完成
n. 板
n. 海豚
n./v. 保證；擔保品
adj. 便利的
n. 心理學
adj./n. 紫色的
v. 創立
adj. 昏昏欲睡的
n. 洗手間；廁所
adj. 不潔的；卑鄙的
n. 配給
